modules/javafx.graphics/src/test/java/test/javafx/scene/CssStyleHelperTest.java

```diff
@@ -39,8 +39,8 @@
 import javafx.scene.text.Text;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
how did I miss that???

Reply:
Worth noting is that PR #1503 also uses JUnit 5 for its new tests, and in a non-trivial manner (not easy to replace). Maybe the best thing to do is to convert this test class to JUnit 5?

Reply:
Maybe we should ask people to convert to junit5 every time they modify a test.

I know we usually ask not to combine different issues in one PR, but given the fact that we are changing the tests there is no (or little) associated risk of regression or merge conflicts.  What do you think?

Reply:
Not as a general policy, but perhaps in some cases. I only suggested it for this class because there are two PRs adding JUnit 5 tests to this class that would benefit from such a conversion, but now that I look at that class more closely, I don't think I'd want to see it as part of either PR.

In general, I don't want to see a JUnit 5 conversions as part of a simple PR that adds a test method to a class with dozens of existing test methods, and this PR falls into that use case.